### PR TITLE
Created new command for creating default categories

### PR DESCRIPTION
**Pull request by core23**
This command is based on https://github.com/sonata-project/SonataClassificationBundle/blob/master/Command/FixContextCommand.php to create default categories defined under ``sonata_media => contexts`` in the configuration.

### CODE REVIEW

**Comment by OskarStark** [comment]
Great work @core23 :+1:

**Comment by core23** [comment]
Travis is failing because there is an issue with mongo extension (not related to this PR).

Anything to do? @OskarStark

**Comment by OskarStark** [comment]
documentation is missing

File: Command/FixMediaContextCommand.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
```
Comment (rande) [inline Command/FixMediaContextCommand.php]: Wrong header ;)

**Comment by core23** [comment]
Added a short documentation.

File: Resources/doc/reference/command_line.rst [hunk 2]
```diff
@@ -93,3 +93,12 @@ The medias.csv file contains the following lines::
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/xqziut_tutoriel-video-symfony-2-twig_lifestyle
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/x9bgxs_php-tv-4-magento-mysql-symfony-zend_tech
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/xhq4c5_slyblog-tutoriel-video-symfony-1-4-partie-2-2_tech
+
+Fix missing default categories
+^^^^^^^^^^^
```
Comment (OskarStark) [inline Resources/doc/reference/command_line.rst]: underline too short

File: Resources/doc/reference/command_line.rst [hunk 3]
```diff
@@ -93,3 +93,12 @@ The medias.csv file contains the following lines::
...
+
+Creates default root category for the ``SonataClassificationBundle`` if they don't exist.
```
Comment (OskarStark) [inline Resources/doc/reference/command_line.rst]: categorieS ?

Comment (core23) [inline Resources/doc/reference/command_line.rst]: Fixed

File: Resources/doc/reference/command_line.rst [hunk 4]
```diff
@@ -93,3 +93,12 @@ The medias.csv file contains the following lines::
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/xqziut_tutoriel-video-symfony-2-twig_lifestyle
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/x9bgxs_php-tv-4-magento-mysql-symfony-zend_tech
    sonata.media.provider.dailymotion,default,http://www.dailymotion.com/video/xhq4c5_slyblog-tutoriel-video-symfony-1-4-partie-2-2_tech
+
+Fix missing root categories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Creates default root categories for the ``SonataClassificationBundle`` if they don't exist.
```
Comment (OskarStark) [inline Resources/doc/reference/command_line.rst]: can you please add a short explanation, WHY this could happen?

Comment (core23) [inline Resources/doc/reference/command_line.rst]: @OskarStark Added a little more information which should explain the command.

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**